modules/ROOT/pages/cloudhub-api.adoc

```diff
@@ -9,27 +9,35 @@ image:logo-hybrid-disabled.png[link="/runtime-manager/deployment-strategies", ti
 image:logo-server-disabled.png[link="/runtime-manager/deployment-strategies", title="Anypoint Platform Private Cloud Edition"]
 image:logo-pcf-disabled.png[link="/runtime-manager/deployment-strategies", title="Pivotal Cloud Foundry"]
 
-The CloudHub REST API provides a way for you to programmatically access many of the things you can do through the Runtime Manager. It allows you to:
+The CloudHub REST API enables you to programmatically access these functions of Runtime Manager:
 
-* Create an application on CloudHub
-* Change the application properties, including the number of workers and environment variables
-* Deploy a new version of your application
-* Delete your application
-* Get statistics about your application
-* Create CloudHub notifications
-* Create e-mail alerts triggered by your applications
+* Create an application on CloudHub.
+* Change the application properties, including the number of workers and environment variables.
+* Deploy a new version of your application.
+* Delete your application.
+* Get statistics about your application.
+* Create CloudHub notifications.
+* Create email alerts triggered by your applications.
 
-Before getting started with specific operations, be sure you're familiar with Operations for Applications.
+For an interactive reference that includes supported resources, methods, required properties, and expected responses, see the https://anypoint.mulesoft.com/exchange/portals/anypoint-platform/f1e97bc6-315a-4490-82a7-23abe036327a.anypoint-platform/cloudhub-api/[CloudHub API Portal].
+
+The CloudHub API applies only to deployments to the cloud-based version of the Runtime Manager. To manage on-prem applications using an API, see the https://anypoint.mulesoft.com/apiplatform/anypoint-platform/#/portals/organizations/ae639f94-da46-42bc-9d51-180ec25cf994/apis/38784/versions/5567732[Runtime Manager API Portal].
```

Comment:
API "applies to" deployments? I think there might be a more precise term for this relationship.

Reply:
```suggestion
The CloudHub API applies only to deployments to the cloud-based version of Runtime Manager. To manage on-prem applications using an API, see https://anypoint.mulesoft.com/apiplatform/anypoint-platform/#/portals/organizations/ae639f94-da46-42bc-9d51-180ec25cf994/apis/38784/versions/5567732[Runtime Manager API Portal].
```